Content.Server/GameObjects/Components/Metabolism/MetabolismComponent.cs

```diff
@@ -356,12 +356,15 @@ private void StopSuffocation()
         {
             Suffocating = false;
 
-            if (Owner.TryGetComponent(out IDamageableComponent? damageable))
+            if (Owner.TryGetComponent(out IDamageableComponent? damageable)
+                && damageable.TryGetDamage(DamageType.Asphyxiation, out var damage)
+                && damage > 0)
             {
                 damageable.ChangeDamage(DamageType.Asphyxiation, -_suffocationDamageRecovery, false);
             }
 
-            if (Owner.TryGetComponent(out ServerAlertsComponent? alertsComponent))
+            if (Owner.TryGetComponent(out ServerAlertsComponent? alertsComponent)
+                && alertsComponent.IsShowingAlert(AlertType.LowOxygen))
```

Comment:
Isn't this already handled by ServerAlertsComponent?
https://github.com/space-wizards/space-station-14/blob/818bdb3c63b30afa1a884dd03fcca102b298d6cd/Content.Shared/GameObjects/Components/Mobs/SharedAlertsComponent.cs#L140-L143